### DISCUSSION TITLE
UsePaging() for ICollections on an ObjectType?

### DISCUSSION THREAD

**Discussion by ghost** [ChilliCream/hotchocolate#4910] [Q&A]
I may be just missing it, so hopefully it's something simple! I have a query setup that is pulling an object successfully using [UseProjections]. One of the fields pulling in is an ICollection of another ObjectType. I would like to add pagination to that ICollection, but I receive an error `[Model] does not have a default constructor (Parameter 'type')`.

Do I need to add something to my field to allow the pagination to work as a sub-field level? Or is this a limitation I'll need to deal with? I'm primarily using this to get the TotalCount for that ICollection.

**Comment by DijanaPenic** [comment]
Might be related to this issue report #4424.
